pkg/virt-controller/watch/application.go

```diff
@@ -632,6 +632,7 @@ func (vca *VirtControllerApp) initCommon() {
 		vca.cdiConfigInformer,
 		vca.clusterConfig,
 		topologyHinter,
+		WithIPAMClaimManager(),
```

Comment:
but then the constructor is not self contained / explicit (raising it because had some discussion similiar)
if you still desire i will do it

iiuc you suggest
```
	vca.vmiController, err = NewVMIController(vca.templateService,
		vca.vmiInformer,
		vca.vmInformer,
		vca.kvPodInformer,
		vca.persistentVolumeClaimInformer,
		vca.storageClassInformer,
		vca.vmiRecorder,
		vca.clientSet,
		vca.dataVolumeInformer,
		vca.storageProfileInformer,
		vca.cdiInformer,
		vca.cdiConfigInformer,
		vca.clusterConfig,
		topologyHinter,
	)
	if err != nil {
		panic(err)
	}

	vca.vmiController.ipamClaimsManager = ipamclaims.NewIPAMClaimsManager(
		vca.vmiController.clientset.NetworkClient(),
		vca.vmiController.clientset.IPAMClaimsClient())
```